### PR TITLE
[Documentation] Jenkins alcor ping test

### PR DESCRIPTION
**Pull request by kiran1048**

### CODE REVIEW

**Review by pkommoju — APPROVED** [review]
Looks good.

File: JENKINS_busybox_ping_test.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+```
+
+# Alcor Ping Test Setup
+Running of automated Alcor builds and ping tests through jenkins can be performed by following these steps:
+* For the Alcor ping tests, ideally we will require 3 machines. For our test case, an ubuntu VM is sufficient. Create and identify the 3 VMs. Lets call them host1, host2 and host3.
+
+* host1 will be used for building and running the alcor services. It will also be the node which will access host2 and host2 to deploy the busybox containers on them and run a ping test between them. See the README.rd file of Alcor to learn about setting up of this host with necessary packages and configuration.
```
Comment (xieus) [inline JENKINS_busybox_ping_test.md:24]: host 2 and host 2 => host 2 and host 3

File: JENKINS_busybox_ping_test.md [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+2. Once on the job page, the left hand column will show many options, one of them would be 'Build Now'. Select it and it will immediately start the job.
+
+3. The newly triggered job will appear under the 'build history' section near the bottom left side of the page. Click on the progressing job to go to the job's page. You can now see a 'Console Output' option in the left column. Clicking here will take you to a page where you can see the command line output of the job. The page keeps flushing out the output until the job is over.
+
+4. At the end a successful job will show up in the history with a blue icon and failure with a red icon. 
+
+
+#
+Miscellaneous
+1. The jenkins workspace is by default located at /var/lib/jenkins/
```
Comment (xieus) [inline JENKINS_busybox_ping_test.md:124]: We will need a "." here :-)

**Review by xieus — APPROVED** [review]
@kiran1048 LGTM. I left two comments which are minor. Pls take a quick look and let us merge. Thanks!

**Review by xieus — APPROVED** [review]

**Pull request merged**